### DISCUSSION TITLE
How do I add my framework in your platform so that it can be tested?

### DISCUSSION THREAD

**Discussion by g0d** [TechEmpower/FrameworkBenchmarks#6846] [Q&A]
https://github.com/g0d/micro-MVC

**Comment by joanhey** [comment]
HI,

I can help you, but first you need to create a repo with the app working based with the requirements:
https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview

You can check the code of other php frameworks:
https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/PHP

Notify me when you have the repo with the app.

Regards
